test/linters_test.go

```diff
@@ -183,10 +183,12 @@ func extractRunContextFromComments(t *testing.T, sourcePath string) *runContext
 	scanner := bufio.NewScanner(f)
 	for scanner.Scan() {
 		line := scanner.Text()
+		if strings.HasPrefix(line, "/*") {
```

Comment:
I'm afraid you have to take more cases into consideration:
* Lines between end and beginning of multi-line comment
* End of multi-line comment

Reply:
Sure, fixed!